docs/reference/search/search.asciidoc

```diff
@@ -302,6 +327,58 @@ If the <<search-api-scroll-query-param,`scroll` parameter>> is specified, this
 value cannot be `0`.
 --
 
+`_source`::
+(Optional)
+Indicates which <<mapping-source-field,source fields>> are returned for matching
+documents. These fields are returned in the `hits._source` property of
+the search response. Defaults to `true`.
++
+.Valid values for `_source`
+[%collapsible%open]
+====
+`true`::
+(boolean)
+The entire document source is returned. This does not include:
+* <<multi-fields,multi-fields>>
+* <<alias,field aliases>>
+* Values copied using the <<copy-to,`copy_to`>> mapping parameter
+
+`false`::
+(boolean)
+The document source is not returned.
+
+`<wildcard_pattern>`::
+(string)
+Wildcard (`*`) patterns containing source fields to return.
+
+`<wildcard_pattern_array>`::
+(array of strings)
+Array of wildcard (`*`) patterns containing source fields to return.
+
+`<object>`::
+(object)
+Object containing a list of source fields to include or exclude.
++
+.Properties for `<object>`
+[%collapsible%open]
+=====
+`includes`::
+(array of strings)
+Array of wildcard (`*`) patterns containing source fields to return.
++
+If this property is specified, only source fields that match these patterns are
+returned. You can exclude fields from this subset using the `excludes` property.
+
+`excludes`::
+(array of strings)
+Array of wildcard (`*`) pattern containing source fields to exclude from the
+response.
++
+If the `includes` property is not specified, the entire document source is returned,
```

Comment:
Similar comment to above, you can use `includes` and `excludes` in conjunction with each other.